### ISSUE TITLE
Explore use of EGL (or GLX-enabled container) with Gazebo headless 

### ISSUE THREAD

**Issue by darknight-007** [schmittlema/openuav-ros-px4-tensorflow#1]
Xvfb (virtual frame buffer) running in containers does not use GLX. Hence camera plugins run at low rendering rate.

**Comment by schmittlema** [comment]
Yeah I ran into this problem for my nvidia container. I had to use a ssh server as the display to get glx working. bit hacky tho
